src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -803,11 +837,648 @@ public void close() {
         waitLogoutResponse();
     }
 
+    // <editor-fold desc="strategies definitions goes here.">
+
+    // <editor-fold desc="send strategies definitions goes here."
+
+    private CompletableFuture<MessageID> defaultSend(IChannel channel, ByteBuf message, Map<String, String> properties, EventID eventID) {
+        return channel.send(message, properties, eventID, SendMode.HANDLE_AND_MANGLE);
+    }
+
+    // FIXME: Should we send batch pieces separately to mstore?
```

Comment:
TBU

Reply:
> has idea to send the buffer with the whole messages to mstore instead of pieces

If we do it any other way a codec won't be able to decode the message at all. So, there is no point in sending separate small buffers...